frontend/src/utilities/constants.ts

```diff
@@ -161,6 +161,776 @@ export const creditOptions = [
   { label: '2', value: 2 },
 ] as const;
 
+export const courseInfoAttributesOptions = [
```

Comment:
Question for the future: is there a way to dynamically get this? Do we really need to statically code every single course attribute? They can easily get stale through time!